docs/bin/lint-md

```diff
@@ -14,11 +14,16 @@ done
 DOCS_DIR="$(cd "$(dirname "$BIN")/.." && pwd)"
 REMARK_RC="$DOCS_DIR/remarkrc"
 
-# This is tricky. `remark` is a dep of mobify-code-style
-# and so won't show up in `node_modules/.bin` of a project
-# that uses mobify-code-style. We have to move into the
-# the mobify-code-style /docs directory and find the npm
-# bin directory relative that _that_ to find remark!
-REMARK_CLI="$(cd "$DOCS_DIR" && npm bin)/remark"
+if [[ $(node -v) =~ ^v[0-4] ]]; then
+	# Node v4!!
+	# This is tricky. `remark` is a dep of mobify-code-style
+	# and so won't show up in `node_modules/.bin` of a project
+	# that uses mobify-code-style. We have to move into the
+	# the mobify-code-style /docs directory and find the npm
+	# bin directory relative that _that_ to find remark!
+    REMARK_CLI="$(cd "$DOCS_DIR" && npm bin)/remark"
```

Comment:
@benlast I'm hoping the v4 branch of this script begins to rot very fast and we can just delete it in the not-to-distant future.  This part of the branch is already working so I'd prefer to leave it if that's ok with you.